bundles/org.eclipse.jface/src/org/eclipse/jface/util/Throttler.java

```diff
@@ -97,13 +97,27 @@ public Throttler(Display display, Duration minWaitTime, Runnable runnable) {
 	 * returning.
 	 */
 	public void throttledExec() {
+		throttledExec(true);
+	}
+
+	/**
+	 * Like {@link #throttledExec()} but guaranteed to not run the Runnable before
+	 * this method returns.
```

Comment:
`guaranteed to not run the Runnable before this method returns.`
The code doesn't match the contract.
One can put breakpoint at line 124 and the code on display will run before the return, assuming current call is not in the UI thread.

Reply:
valid point. i fixed the javadoc